apps/admin/code/package.json

```diff
@@ -52,7 +52,9 @@
   "scripts": {
     "analyze": "source-map-explorer build/static/js/main.*",
     "watch": "cross-env PORT=3001 yarn webiny run watch",
-    "build": "yarn webiny run build"
+    "build": "yarn webiny run build",
+    "build:dev": "yarn webiny run build --env dev",
```

Comment:
Let's remove these custom scripts. No need to add additional stuff we need to explain. If users will need these commands, they can add them themselves.